vllm/entrypoints/openai/serving_chat.py

```diff
@@ -175,26 +218,45 @@ def _parse_chat_message_content_parts(
                                       image_futures=image_futures)
 
     def _parse_chat_message_content(
-        self,
-        message: ChatCompletionMessageParam,
+            self,
+            message: ChatCompletionMessageParam,
     ) -> ChatMessageParseResult:
-        role = message["role"]
+        role = message.get('role')
         content = message.get("content")
+        tool_call_id = message.get('tool_call_id')
+        tool_calls = message.get('tool_calls')
+        name = message.get('tool_calls')
```

Comment:
Shouldn't this be

```python
name = message.get('name')
```

?

Reply:
Yes, great catch. thanks!

Reply:
4356ec4dde3f544020a674f1f65a91710feb319f